cpg_workflows/stages/large_cohort.py

```diff
@@ -489,6 +490,195 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
         return self.make_outputs(cohort, data=self.expected_outputs(cohort), jobs=[j])
 
 
+@stage(required_stages=[Combiner])
+class ShardVds(CohortStage):
```

Comment:
Is this important? Query on Batch already parallelises all operations by partition, so explicitly pre-sharding this object shouldn't do much, but will double the data we store in GCP

Reply:
This workflow shards at one end, and merges at the other, but QOB already provides parallelised operations :/